blocks/double-chain-block/chains/double-chain/default.jsx

```diff
@@ -13,27 +13,25 @@ const DoubleChain = ({ children, customFields }) => {
 		// check column one length not negative
 		if (columnOneLength > 0) {
 			const childrenOutput = (
-				<div className="container-fluid double-chain chain-container">
-					<div className="row wrap-bottom">
-						<div className="col-sm-12 col-md-xl-6 ie-flex-100-percent-sm column-1 reduce-internal-row-col-gap chain-col">
-							{children.slice(0, columnOneLength)}
-						</div>
-						<div className="col-sm-12 col-md-xl-6 ie-flex-100-percent-sm column-2 reduce-internal-row-col-gap chain-col">
-							{children.slice(columnOneLength)}
-						</div>
-					</div>
-				</div>
+				<Grid className={`${BLOCK_CLASS_NAME}__children-grid`}>
+					<Stack direction="vertical" className={`${BLOCK_CLASS_NAME}__child-item`}>
+						{children.slice(0, columnOneLength)}
+					</Stack>
+					<Stack direction="vertical" className={`${BLOCK_CLASS_NAME}__child-item`}>
+						{children.slice(columnOneLength)}
+					</Stack>
+				</Grid>
 			);
 
-			if (!heading) {
-				return childrenOutput;
-			}
-
 			return (
-				<HeadingSection>
-					<Heading className="chain-heading">{heading}</Heading>
-					{childrenOutput}
-				</HeadingSection>
+				<Stack direction="vertical" className={BLOCK_CLASS_NAME}>
+					<HeadingSection>
+						{heading ? (
+							<Heading className={`${BLOCK_CLASS_NAME}__heading`}>{heading}</Heading>
+						) : null}
+						{childrenOutput}
+					</HeadingSection>
+				</Stack>
```

Comment:
```suggestion
				<Stack direction="vertical" className={BLOCK_CLASS_NAME}>
					{heading ? (
						<HeadingSection>
							<Heading className={`${BLOCK_CLASS_NAME}__heading`}>{heading}</Heading>
						</HeadingSection>
					) : null}
					{childrenOutput}
				</Stack>
```

Reply:
Heading section increments the header, and shouldn't do that if the header isn't shown.  This feels like it might be a bug in the original.